vantage/team_resource.go

```diff
@@ -13,6 +14,12 @@ import (
 	teamsv2 "github.com/vantage-sh/vantage-go/vantagev2/vantage/teams"
 )
 
+var (
+	_ resource.Resource                = (*TeamResource)(nil)
```

Comment:
nit - could use the `&TeamResource{}` form for consistency with the other changes of this kind

Reply:
🤔 this was the change i made. i wonder if theres a linter thats swapping it back in some cases. will get them all to be consistent

Reply:
thanks, updated them all to match. swapped back to the `(*Resource)(nil)` pattern to keep the robots happy.